scripts/machine-launch-script.sh

```diff
@@ -74,12 +74,21 @@ sudo python3 -m pip install moto==2.2.17
 sudo python3 -m pip install sure==2.0.0
 # needed for the awstools cmdline parsing
 sudo python3 -m pip install pyyaml==5.4.1
+sudo python3 -m pip install mypy==0.931
+sudo python3 -m pip install types-PyYAML==6.0.4
+sudo python3 -m pip install boto3-stubs==1.21.6
+sudo python3 -m pip install botocore-stubs==1.24.6
+sudo python3 -m pip install mypy-boto3-ec2==1.21.1
+sudo python3 -m pip install mypy-boto3-s3==1.21.0
 
 # setup argcomplete
 activate-global-python-argcomplete
 
 } 2>&1 | tee /home/centos/machine-launchstatus.log
 
+# add python installs to PATH
+echo "export PATH=/usr/local/bin:\$PATH" >> /home/centos/.bashrc
+
```

Comment:
I don't understand why this needs to be added.  I see `/usr/local/bin` in the centos user's path on a fresh FPGA Dev AMI instance.

Reply:
I think this change was done to support Ubuntu. I'll move this to the Ubuntu script